includes/gateways/stripe/includes/payment-methods/class-give-stripe-becs.php

```diff
@@ -192,7 +192,7 @@ class="give-stripe-becs-bank-account-field give-stripe-cc-field"
 		 * @param array $donation_data List of donation data.
 		 *
 		 * @return void
-		 * @since  2.6.3
+		 * @since  2.7.0
```

Comment:
If this has existed since 2.6.3, this shouldn't be updated.

Reply:
@bradp This BECS Direct Debit related functionality is not released and will get released as a part of 2.7.0. So, I have updated it to avoid confusion in the future.